authorization-dotnet/EncryptingAndDecryptingStringInCSharp/EncryptingAndDecryptingStringInCSharp/StringEncryptionService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Security.Cryptography;
+using System.Text;
+
+namespace EncryptingAndDecryptingStringInCSharp
+{
+    public class StringEncryptionService
+    {
+        private byte[] IV =
+            {
+                0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
+                0x09, 0x10, 0x11, 0x12, 0x13, 0x14, 0x15, 0x16
+            };
```

Comment:
Please fix the indentation, curly braces should be right below the letter "p" of the `private` keyword, same as you did for the methods in this class :)

Reply:
fixed